records/models.py

```diff
@@ -57,9 +58,67 @@ class Record(models.Model):
     transaction_type = models.CharField(choices=TRANSACTION_TYPE, max_length=3)
     created_at = models.DateTimeField(auto_now_add=True)
 
+    @property
+    def redis_tags_key(self):
+        '''
+            Return Redis key where stored sorted set of tags frequency usage.
+        '''
+        return settings.REDIS_KEY_USER_TAGS % (self.user_id,)
```

Comment:
we have too many duplication of code like `REDIS_KEY_USER_TAGS % user_id`. Please fix this duplication. ([DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself))